map/src/main/kotlin/edu/artic/map/MapFragment.kt

```diff
@@ -239,6 +237,33 @@ class MapFragment : BaseViewModelFragment<MapViewModel>() {
                 return@setOnMarkerClickListener true
             }
             groundOverlayGenerated.onNext(true)
+
+            /**
+             * Add Lion icons.
+             * TODO:: fix bounds of map.
+             */
+            map.addMarker(MarkerOptions()
+                    .position(LatLng(41.879678006591391, -87.624091248446064))
```

Comment:
I think harcoding the position and `MarkerOptions` for these lions are problematic from a code maintainability and class file length perspective. I think a helper function that returns these lion 1 + lion 2 will be more readable. Also, do these _always_ display in any mode at all zoom levels? if not I will add them in my branch where its possible to do that.